changelog.d/3184.feature

```diff
@@ -0,0 +1 @@
+Add /_media/r0/config
```

Comment:
I guess this won't mean much to the average reader of the synapse changelog. You probably need to say it's a new endpoint for clients. A link to the proposal might help.